src/hooks/useCioAutocomplete.ts

```diff
@@ -212,30 +212,28 @@ const useCioAutocomplete = (options: UseCioAutocompleteOptions) => {
       'data-testid': 'cio-form',
     }),
     getSectionProps: (section: Section) => {
-      const { type, displayName } = section;
-      let sectionTitle = displayName;
+      const { type } = section;
+      let sectionTitle: string;
```

Comment:
Can we rename this to sectionClassName instead? or anything that says this differs from the section's actual title? The name is confusing and coming back to this in the future we would forget that this is only for the class name

Reply:
I'm in total agreement. @esezen I think you worked on this previously, but would `sectionClassName` work for you as well?

Reply:
We'll track this on PR #128 instead